ompi/mca/coll/tuned/coll_tuned.h

```diff
@@ -148,6 +150,7 @@ int ompi_coll_tuned_gather_intra_check_forced_init (coll_tuned_force_algorithm_m
 
 /* Reduce */
 int ompi_coll_tuned_reduce_intra_dec_fixed(REDUCE_ARGS);
+int ompi_coll_tuned_reduce_intra_sm_dec_fixed(REDUCE_ARGS);
```

Comment:
Maybe we should use a better name than `sm`.

Reply:
Mhh, I wanted to use `intra` but that is used for intra-communicators already and `ompi_coll_tuned_reduce_intra_intra_dec_fixed` is weird. Other ideas to replace `sm`: `singlenode`, `sharedmem`, `local`

Reply:
1node, single_node. I actually like local, but it might be confusing for some.

Reply:
I went with `_singlenode_`